vendor/modules.txt

```diff
@@ -1332,6 +1332,7 @@ github.com/portworx/sched-ops/k8s/errors
 github.com/portworx/sched-ops/k8s/externalsnapshotter
 github.com/portworx/sched-ops/k8s/externalstorage
 github.com/portworx/sched-ops/k8s/kubevirt
+github.com/portworx/sched-ops/k8s/kubevirt-dynamic
```

Comment:
This is fine, but it might be easier to use the typed client using kubevirt.GetKubeVirtClient().

Reply:
Will work on this..

Reply:
we will upgrade later, as of now its ok.